<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="offset_x_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="width_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="offset_y_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="exp_time">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure time (ms)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
